app/js/lib/server/index.js

```diff
@@ -21,7 +21,7 @@ const p = Object.freeze({
 
 export default class Server extends EventDispatcher {
   constructor({ settings, net } = {}) {
-    super(['online', 'push-message']);
+    super(['logged-in', 'online', 'push-message']);
```

Comment:
Can you call it `login`? It's better to use words consistently rather than starting using verbs.